WcaOnRails/config/locales/en.yml

```diff
@@ -371,7 +371,7 @@ en:
       gender:
         male: "Male"
         female: "Female"
-        other: "Other"
+        other_gender: "Other"
```

Comment:
Did this have to change because "other" is a special word for pluralization purposes?